src/components/MerchantsPage/ContributionBar.tsx

```diff
@@ -1,20 +1,23 @@
 import * as React from 'react';
 import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
 
 interface Props {
   totalDonations: number;
   totalGiftCards: number;
 }
 
 const ContributionBar = ({ totalDonations, totalGiftCards }: Props) => {
+  const { t } = useTranslation();
```

Comment:
I wonder if its possible to declare { t } in App.tsx so that its available to the whole app by default...(just an idea, may or may not be possible)

Reply:
I'm fairly new to React, but I think the only ways to achieve something like this would be to pass it from the App component down through to child components, or to set it as context. I don't think either are cleaner than this though--what do you think?